lib/Collection.js

```diff
@@ -69,17 +69,17 @@
 					callback(new Error('The term parameter must be supplied for finding ' + kind + '.'));
 					return false;
 				}
-				return restHandlers.findItems(kind, params, function(error, data, additionalData, req, res) {
+				return restHandlers.findItems(kind, params, function(error, data, additionalData, req, res, relatedObjects) {
 					var collectionItems = wrapCollectionItems(data, kind, options);
-					callback(error, collectionItems, additionalData, req, res);
+					callback(error, collectionItems, additionalData, req, res, relatedObjects);
```

Comment:
Tried using .find on **deals, users and persons**. Looks like they don't have related_objects in response, so it might not be needed, but for consistency and future-proof I think it should stay